docs/make.jl

```diff
@@ -4,6 +4,7 @@ makedocs(
     modules = [ContMechTensors],
     format = :html,
     sitename = "ContMechTensors.jl",
+    doctest = true,
```

Comment:
Yes, but I wanted to turn them off for a while so I had it to false. It doesn't hurt to be explicit.